File: MasterCatalog.Domain/Models/ItemImage.cs
namespace MasterCatalog.Domain.Models
{
    public class ItemImage
    {
        public int? ItemImageID { get; set; }

        public int ItemID {  get; set; }

        public string? AbsoluteUri { get; set; }

        public int DisplayOrder { get; set; }
    }
}

File: MasterCatalog.Web/wwwroot/js/models/itemImage.js
export class ItemImage {
    itemImageID;
    itemID;
    absoluteUri;
    displayOrder;

    constructor(id, itemID, absoluteUri, displayOrder) {
        this.itemImageID = id;
        this.itemID = itemID;
        this.absoluteUri = absoluteUri;
        this.displayOrder = displayOrder;
    }
}